tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/selection-start-end.html

```diff
@@ -59,10 +59,18 @@
     assert_equals(testValue.length, 10);
   }, "Sanity check for testValue length; if this fails, variou absolute offsets in the test below need to be adjusted to be less than testValue.length");
 
+  for (let prop of ["selectionStart", "selectionEnd"]) {
+    test(function() {
```

Comment:
This outer test is not necessary.

Reply:
Thanks. I thought that maybe it was a bad idea to call `createTestElements` outside of a `test`, but perhaps it doesn't actually matter.